### PR TITLE
chore: improved logging

### PR DESCRIPTION
**Pull request by mbruzda-splunk**

### CODE REVIEW

**Comment by artemrys** [comment]
Tests are failing.

**Review by artemrys — DISMISSED** [review]

File: main.py [hunk 1]
```diff
@@ -5,12 +5,15 @@
...
 import json
 import yaml
+import logging
 
 from pathlib import Path
 from typing import Dict, Any, Tuple, Callable, Sequence, Optional, List
 
 NUM_RETRIES = 3
 
+logging.basicConfig(level=logging.INFO)
```
Comment (mkolasinski-splunk) [inline main.py:15]: debug logs will be missed with this configuration of logging.
Consider changing to debug or adding a parameter to control level of logging

Comment (artemrys) [reply main.py:15]: let's have a debug logging by default for the beginning, after some time we can switch to info
and +1 for adding parameter to control the level

**Review by mkolasinski-splunk — DISMISSED** [review]

**Comment by mbruzda-splunk** [comment]
tested here: 
https://github.com/splunk/splunk-add-on-for-cisco-meraki/actions/runs/5679592282/job/15392114852?pr=402

**Comment by mbruzda-splunk** [comment]
https://github.com/splunk/splunk-add-on-for-cisco-meraki/actions/runs/5724604274/job/15511516302

**Review by mkolasinski-splunk — APPROVED** [review]

**Review by mkolasinski-splunk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ jobs:
 | `app_path`      | Path to the directory where addon is located, without filename                 | **required** |         |
 | `included_tags` | Comma separated list of [tags](#reference-docs) to include in appinspect job   |              | None    |
 | `excluded_tags` | Comma separated list of [tags](#reference-docs) to exclude from appinspect job |              | None    |
+| `log_level`     | Python logging level for action                                                |              | `INFO   |
```
[the remark this answers is not in the document]
Comment (mbruzda-splunk) [reply README.md]: fixed

**Review by mkolasinski-splunk — APPROVED** [review]

**Pull request merged**